src/components/icons/Spinner/Spinner.component.tsx

```diff
@@ -0,0 +1,7 @@
+import { Loader } from 'lucide-react';
+
+import { cn } from '@/lib';
+
+export const Spinner = ({ ...props }: React.ComponentProps<'svg'>) => {
+  return <Loader {...props} className={cn('animate-spin')} />;
```

Comment:
remove the function. Lets focus of performance. Of-course it isn't much, but its there